.travis.yml

```diff
@@ -199,7 +199,7 @@ jobs:
         - bash ./scripts/docker-run-memory-test.sh
 
     # Sanitizer tests
-    - if: (type = cron) OR (branch = prerelease_test)
+    - if: (type = cron) OR NOT (branch = prerelease_test)
```

Comment:
why are we not running this in prerelease now?

Reply:
this was just to get it to run on the PR will revert .travis.yml change before merging